docs/asciidoc/modules/ROOT/examples/generated-documentation/apoc.periodic.countdown.adoc

```diff
@@ -1,5 +1,5 @@
 ¦xref::overview/apoc.periodic/apoc.periodic.countdown.adoc[apoc.periodic.countdown icon:book[]] +
 
-`apoc.periodic.countdown('name',statement,repeat-rate-in-seconds) submit a repeatedly-called background statement until it returns 0`
+`apoc.periodic.countdown(name,statement,repeat-rate-in-seconds)` creates a background job that will repeatedly execute the given Cypher statement until it returns 0`
```

Comment:
Yes, to change these parts you would need to change the value of the strings defined in `@Description` annotation (as done with the `apoc.periodic.repeat`), and then run the test `DocsTest` test which would change all descriptions.

However, the `DocsTest` regenerates all procedure files, but for some procedures, for example [these ones](https://neo4j.com/labs/apoc/4.1/mathematical/sigmoid-hyperbolic-operations/), we have chosen to create an ad hoc file instead to create a lot of generated new files.
So we should rollback all the parts you don't need (in this example all except `apoc.periodic.repeat/countdown/list`).

In case, I could do this if you want.

The problem is that in this case the `Name | Type | Description` modification would be reset (see comment below).